src/frontend/src/pages/AdminToolsPage/RecruitmentConfig/FAQTable.tsx

```diff
@@ -1,25 +1,42 @@
+import React, { useState } from 'react';
 import { TableRow, TableCell, Box, Table as MuiTable, TableHead, TableBody, Typography, Button } from '@mui/material';
 import EditIcon from '@mui/icons-material/Edit';
 import DeleteIcon from '@mui/icons-material/Delete';
 import { FrequentlyAskedQuestion } from 'shared/src/types/frequently-asked-questions-types';
 import { NERButton } from '../../../components/NERButton';
-import { useAllFaqs } from '../../../hooks/recruitment.hooks';
+import { useAllFaqs, useDeleteFAQ } from '../../../hooks/recruitment.hooks';
 import LoadingIndicator from '../../../components/LoadingIndicator';
 import { useHistoryState } from '../../../hooks/misc.hooks';
 import ErrorPage from '../../ErrorPage';
 import CreateFaqFormModal from './CreateFaqFormModal';
 import EditFaqFormModal from './EditFaqFormModal';
+import NERDeleteModal from '../../../components/NERDeleteModal';
+import { useToast } from '../../../hooks/toasts.hooks';
 
 const FAQsTable = () => {
   const [createModalShow, setCreateModalShow] = useHistoryState<boolean>('', false);
   const [faqEditing, setFaqEditing] = useHistoryState<FrequentlyAskedQuestion | undefined>('', undefined);
+  const [faqToDelete, setFaqToDelete] = useState<FrequentlyAskedQuestion | undefined>(undefined);
+  const [deleteModalShow, setDeleteModalShow] = useHistoryState<boolean>('', false);
```

Comment:
you don't need the delete modal show, you can just use the faqToDelete to check if its not undefined